genesiscommunitysuccess/custom-elements-lsp
# fix: remove source files from dist bundle FUI-1516

🤔  &nbsp; **What does this PR do?**

- Fix the location of the `.npmignore` which ensures that only the required dist files are included in the registry bundle

📑  &nbsp; **How should this be tested?**

Update test steps to match your PR:

```
1. Checkout branch
2. `npm run bootstrap`
3. `npm run test:unit`
Check the bundled files
1. `cd packages/core/custom-elements-lsp`
2. `npm publish --dry-run`
3. Verify that the only included files are from the `out` and `parser` directories (and the licence and readme which are always included)
```

✅  &nbsp; **Checklist**
